### PR TITLE
Add colored indicator to timeline

### PR DESCRIPTION
**Pull request by dave-v0**
## Purpose

The timeline of the ScholarX page does not have an indicator to specify the current timeline date #1202 

## Goals
To add a colored indicator to the timeline of the page

## Approach
Added color to the present timeline date of the ScholarX process

### Screenshots
image

  
### Preview Link

https://pr-{PR_NUMBER}-sef-site.surge.sh/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
```
Comment (anjula-sack) [inline .vscode/settings.json]: Remove this file

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Good job! @DilinaBandara Can you remove that unwanted file?

File: .vscode/settings.json (hunk 1 above)
Comment (dave-v0) [reply .vscode/settings.json]: @anjula-sack Thanks, I removed it

**Review by anjula-sack — APPROVED** [review]
Good job! @DilinaBandara

**Pull request merged**